util.go

```diff
@@ -13,3 +16,35 @@ func dividedOutput(cmd *exec.Cmd) (stdout []byte, stderr []byte, err error) {
 	err = cmd.Run()
 	return outb.Bytes(), errb.Bytes(), err
 }
+
+func newCmd(dir string, env map[string]string, name string, cmd ...string) *exec.Cmd {
+	c := exec.Command(name, cmd...)
+	if len(dir) != 0 {
+		c.Dir = dir
+	}
+	dstEnv := osutil.Environ(os.Environ())
+	for k, v := range env {
+		dstEnv.Set(k, v)
+	}
+	dstEnv.Set("LANG", "C.UTF8")
+	dstEnv.Set("LANGUAGE", "C.UTF8")
```

Comment:
Do both `LANG` and `LANGUAGE` need to be set to fix this issue? If we don't set one of them, will the issue persist?

Why is the value `C.UTF8`, expected it to be `en_US.UTF8` from #6. What does "C" stand for?